src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,122 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { getMovies } from '../../api/api';
+import { MovieCard } from '../MovieCard';
+
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
-
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
-
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
-        </div>
-
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
-        </div>
-      </div>
-    </form>
-
-    <div className="container">
-      <h2 className="title">Preview</h2>
-      {/* <MovieCard  /> */}
-    </div>
-  </>
+type Props = {
+  addMovie: (Movie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = React.memo(
+  ({ addMovie }) => {
+    const [Movie, setMovie] = useState<Movie>();
+
+    const [currentTitle, setCurrentTitle] = useState('');
+    const [title, setTitle] = useState('');
```

Comment:
```suggestion
    const [currentTitle, setCurrentTitle] = useState('');
    const [title, setTitle] = useState('');
```
Не дуже зрозуміло, для чого потрібні дві однакових змінних в стейті